.golangci.yml

```diff
@@ -1,5 +1,5 @@
 linters:
-  disable-all: false                    # This enabled 7 default linters as of golangci-lint@v1.50.0:
+  disable-all: true                    # This enabled 7 default linters as of golangci-lint@v1.50.0:
```

Comment:
Isn't it disabling the default linters? Why do we need this change?

Reply:
https://golangci-lint.run/usage/linters/#enabled-by-default

Reply:
yes fixing this.